tutorials/advanced_features/Output Modulation and EOM Mode.ipynb

```diff
@@ -198,7 +200,7 @@
     "1. Not only the amplitude but also the detuning and phase are modulated, all with the same modulation bandwidth.\n",
     "2. Alignment between channels takes into account the extended duration of the pulses in the other channels. Note, for instance, how the last pulse on the `rydberg` channel starts only after the output of the `raman` channel goes to zero.\n",
     "3. Similarly, changing the target in a local channel will also wait for the output to ramp down before starting the retargeting.\n",
-    "4. For consecutive pulses in the same channel, there is no automatically imposed delay between them to allow one pulse to finish before the next one starts. As such, whenever the interval between two pulses is too short, they will be \"merged\" together, as is illustrated in the `raman` channel."
+    "4. For consecutive pulses in the same channel, there is no automatically imposed delay between them to allow one pulse to finish before the next one starts. As such, whenever the interval between two pulses is too short, they will be \"merged\" together, as is illustrated in the `raman` channel. To avoid this merging, we can set `at_rest=True` in the `delay()` call. As shown above, when this option is combined with an applied delay time of 0 the channel will be delayed by the exact ammount necessary for the the previous pulse to finish."
```

Comment:
```suggestion
    "4. For consecutive pulses in the same channel, there is no automatically imposed delay between them to allow one pulse to finish before the next one starts. As such, whenever the interval between two pulses is too short, they will be \"merged\" together, as is illustrated in the `raman` channel. To avoid this merging, we can set `at_rest=True` in the `delay()` call. As shown above, when this option is combined with an applied delay time of 0 the channel will be delayed by the exact ammount necessary for the previous pulse to finish."
```